source/includes/concept/_geostore.md

```diff
@@ -0,0 +1,50 @@
+## Geostore
```

Comment:
Thanks for the feedback Ethan :bow: 

- Reworded the final paragraphs of the concept docs and tried to simplify the text a bit so that the overall tone is less formal.
- Replaced "Geostore microservice" with "Geostore API".
- Added a section "What are geostores used for?" at the end of the concept docs.